src/components/UI/Overlay/OverlayContent/AddressBook.tsx

```diff
@@ -78,6 +79,7 @@ export const AddressBook = styled(({ onAddressSelected, ...props }: AddressBookP
 
   const queryEndpoint = useCallback(
     debounce(async (search) => {
+      if (!entityLookupPath) throw new Error(`entityLookup feature is not available`);
```

Comment:
Actually, we can prevent the user from navigating here to start with right? If there is no search feature they shouldn;t be able to type anything, or even earlier, like selecting this endpoint to search with. 

I think that can be added in as another PR though, what do you think?

Reply:
yes i agree, will create a user story for this https://www.pivotaltracker.com/story/show/175433574